inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/InlongStreamController.java

```diff
@@ -183,4 +191,22 @@ public Response<List<StreamField>> parseFields(@RequestBody ParseFieldRequest pa
         return Response.success(streamService.parseFields(parseFieldRequest));
     }
 
+    @RequestMapping(value = "/stream/fieldsImportTemplate", method = RequestMethod.GET)
+    @ApiOperation(value = "Download fields import template")
+    public void downloadFieldsImportTemplate(HttpServletResponse response) {
```

Comment:
Suggest return true of false for the API method, you can wrap it with `Response`.